### PR TITLE
Feature/cimms decoder page

### PR DESCRIPTION
**Pull request by jacob6838**
# PR Details

## Description

Adding the CIMMS ASN.1 decoder page to the CVManager Webapp. This page allows users to enter or upload asn.1 encoded messages, passes them on to the conflictvisualizer api to decide, and enables users to visualize this information on a map, as well as download the decoded messages.

## How Has This Been Tested?

Testing was completed locally, using the conflictvisualizer api

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] My changes require new environment variables:
  - [ ] I have updated the docker-compose, K8s YAML, and all dependent deployment configuration files.
- [ ] My changes require updates to the documentation:
  - [ ] I have updated the documentation accordingly.
- [ ] My changes require updates and/or additions to the unit tests:
  - [ ] I have modified/added tests to cover my changes.
- [x] All existing tests pass.

### CODE REVIEW

**Review by drewjj — CHANGES_REQUESTED** [review]
When I try to select a file to decode it looks like it only takes a ZIP file. When I select a file that isn't a ZIP it errors out about it not being a ZIP file. We probably don't want it erroring out like this if a wrong file type is chosen but John is telling me a plain text file is all it should be expecting.

We should probably make it more clear what the kind of data it is expecting.

**Comment by drewjj** [comment]
image

**Review by drewjj — APPROVED** [review]
Nice, I really like that you combined it with the intersection map page. The enabling of the decoder mode also minimizes (in addition to disabling) the other sections which is awesome. It functionally works as expected with visualizing the ASN1.

**Comment by payneBrandon** [comment]
@jacob6838 looks like a merge conflict

**Comment by jacob6838** [comment]
> @jacob6838 looks like a merge conflict

Resolved!

File: webapp/src/generalSlices/intersectionSlice.tsx [hunk 1]
```diff
@@ -23,7 +23,15 @@ export const intersectionMapLabelsLayer: SymbolLayer = {
 }
 
 export const initialState = {
-  intersections: [] as IntersectionReferenceData[],
+  intersections: [
```
Comment (payneBrandon) [inline webapp/src/generalSlices/intersectionSlice.tsx:26]: Is this dummy intersection needed?

Comment (jacob6838) [reply webapp/src/generalSlices/intersectionSlice.tsx:26]: Yeah it is, it's the "-1" intersection, or "no intersection selected". It is used as the default across the cimms components. We use it in the configuration page to see global parameters

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**